From 11cb34978d15b53b16066b7e53fedf9b184f9dbd Mon Sep 17 00:00:00 2001
From: Geliang Tang <geliang.tang@suse.com>
Date: Fri, 26 Aug 2022 09:05:54 +0800
Subject: [PATCH] Revert "Linux 5.10.138"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit fa3303d70b423dd3e855f57febaba77e15069650.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 234c8032c2b4..ac56b4757a6e 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 138
+SUBLEVEL = 137
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.34.1

